kolena/_utils/batched_load.py

```diff
@@ -49,7 +49,7 @@ def init_upload() -> API.InitiateUploadResponse:
 
 def upload_data_frame(df: pd.DataFrame, batch_size: int, load_uuid: str) -> None:
     num_chunks = math.ceil(len(df) / batch_size)
-    chunk_iter = np.array_split(df, num_chunks) if num_chunks > 0 else []
+    chunk_iter = np.array_split(df, num_chunks) if num_chunks > 1 else [df] if num_chunks > 0 else []
```

Comment:
We're still going to get the deprecation warnings when num_chunks > 1. Thinking we should create a ticket to track moving away from `np.array_split` entirely for dataframes